themes/nord-color-theme.json

```
@@ -1215,6 +1215,45 @@
         "foreground": "#81A1C1"
       }
     },
+    {
+      "name": "[Rust] Entity types",
+      "scope": [
+        "source.rust entity.name.type"
+      ],
+      "settings": {
+        "foreground": "#8FBCBB"
+      }
+    },
+    {
+      "name": "[Rust] Macro",
+      "scope": [
+        "meta.macro.rust"
+      ],
+      "settings": {
+        "fontStyle": "bold",
+        "foreground": "#88C0D0"
+      }
+    },
+    {
+      "name": "[Rust] Attributes",
+      "scope": [
+        "meta.attribute.rust",
+        "meta.attribute.rust punctuation",
+        "meta.attribute.rust keyword.operator"
+      ],
+      "settings": {
+        "foreground": "#5E81AC",
+      }
+    },
+    {
+      "name": "[Rust] Interpolation Bracket Curly",
+      "scope": [
+        "punctuation.definition.interpolation.rust"
+      ],
```

```suggestion
      "scope": "source.rust punctuation.definition.interpolation",
```

Same again for the `source.rust` selector and string typing like above.